AK/StringView.h

```diff
@@ -365,6 +365,8 @@ class StringView {
 
 template<>
 struct Traits<StringView> : public DefaultTraits<StringView> {
+    using PeekType = StringView;
+    using ConstPeekType = StringView;
```

Comment:
Shouldn't this be `const` somehow? Is that even possible when not returning a reference or pointer type?

Reply:
It seems to me that StringView (just from the definition, but also by how it's used) is an immutable type in and of itself? Also, such a change requires a bunch of follow-on changes on anyone who returns a StringView from a const member function.